pages/search/advanced.tsx

```diff
@@ -114,6 +114,7 @@ export default function AdvancedSearch({ isAuthenticated }) {
   const handleClear = (e: SyntheticEvent) => {
     e.preventDefault()
     alert && setAlert(false)
+    dateInputRefs.map((ref) => (ref.current.value = ""))
```

Comment:
This doesn't have to be part of this PR, but what if `useDateForm` exposes an additional "clear" function that internally calls `inputRefs.forEach((ref) => (ref.current.value = ""))`? That way the function can be called instead of manually doing it here.

Using `forEach` instead of `map` since `map` generates a new array and we don't need that.

Reply:
i included this update! thanks for the recommendation.

Reply:
Looks good.